src/fides/api/models/privacy_experience.py

```diff
@@ -28,32 +28,32 @@ class ComponentType(Enum):
     privacy_center = "privacy_center"
 
 
-class DeliveryMechanism(Enum):
+class BannerEnabled(Enum):
     """
-    The delivery mechanism - not formalized in the db
+    Whether the banner should display
     """
 
-    banner = "banner"
-    link = "link"
+    always_enabled = "always_enabled"
+    enabled_where_required = "enabled_where_required"  # If the user's region has at least one opt-in or notice-only notice
+    always_disabled = "always_disabled"
```

Comment:
A primary change of this PR; `delivery_mechanism` has been replaced in favor of `banner_enabled` which is similar in concept, however this construct further reduces the connections between a Notice and an Experience.    Previously, we did a lot of behind the scenes validation to make sure an experience's delivery_mechanism always matched its notices.  This new field doesn't have that validation, we don't check this at all, so it's possible `ExperienceConfig.banner_enabled` = `always_disabled` regardless of how the underlying notices should be displayed.